### PR TITLE
ports to libmain

### PR DESCRIPTION
**Pull request by guerinoni**

### CODE REVIEW

File: Userland/Utilities/beep.cpp [hunk 1]
```diff
@@ -1,12 +1,13 @@
 /*
- * Copyright (c) 2020, the SerenityOS developers.
+ * Copyright (c) 2022, the SerenityOS developers.
```
Comment (creator1creeper1) [inline Userland/Utilities/beep.cpp:2]: `2020-2022` would be better

File: Userland/Utilities/tac.cpp [hunk 2]
```diff
@@ -1,27 +1,26 @@
 /*
- * Copyright (c) 2021, Federico Guerinoni <guerinoni.federico@gmail.com>
+ * Copyright (c) 2022, Federico Guerinoni <guerinoni.federico@gmail.com>
```
Comment (creator1creeper1) [inline Userland/Utilities/tac.cpp:2]: `2021-2022` would be better

File: Userland/Utilities/wasm.cpp [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021, Ali Mohammad Pur <mpfard@serenityos.org>
+ * Copyright (c) 2022, the SerenityOS developers.
```
Comment (creator1creeper1) [inline Userland/Utilities/wasm.cpp:2]: Removing someone's name from the copyright header isn't very nice. The following would be better:
```
 * Copyright (c) 2021-2022, Ali Mohammad Pur <mpfard@serenityos.org>
 * Copyright (c) 2022, the SerenityOS developers.
```

**Review by creator1creeper1 — CHANGES_REQUESTED** [review]

File: Userland/Utilities/wasm.cpp (hunk 3 above)
Comment (guerinoni) [reply Userland/Utilities/wasm.cpp:3]: Fixed

File: Userland/Utilities/tac.cpp (hunk 2 above)
Comment (guerinoni) [reply Userland/Utilities/tac.cpp]: Fixed

File: Userland/Utilities/beep.cpp (hunk 1 above)
Comment (guerinoni) [reply Userland/Utilities/beep.cpp]: Fixed

**Comment by creator1creeper1** [comment]
You can mark my requests as fixed by clicking on "resolve conversion".

**Review by creator1creeper1 — APPROVED** [review]

File: Userland/Utilities/wasm.cpp (hunk 3 above)
Comment (bgianfo) [reply Userland/Utilities/wasm.cpp:3]: Why are we changing the date range of @alimpfard's copyright?

**Review by bgianfo — CHANGES_REQUESTED** [review]
Please undo the changes to copyright lines that don't pertain to you.

File: Userland/Utilities/wasm.cpp (hunk 3 above)
Comment (guerinoni) [reply Userland/Utilities/wasm.cpp:3]: Just keep `Copyright (c) 2021, Ali Mohammad Pur <mpfard@serenityos.org>` ?

**Review by bgianfo — APPROVED** [review]

**Pull request merged**